geoapi/services/features.py

```diff
@@ -234,6 +238,47 @@ def fromShapefile(projectId: int, fileObj: IO, metadata: Dict, additional_files:
         db_session.commit()
         return features
 
+    @staticmethod
+    def fromRAPP(projectId: int, fileObj: IO, metadata: Dict, original_path: str = None) -> Feature:
+        """
+
+        :param projectId: int
+        :param fileObj: file descriptor
+        :param metadata: Dict of <key, val> pairs
+        :param original_path: str path of original file location
+        :return: Feature
+        """
+        data = json.loads(fileObj.read())
+
+        lng = data.get('geolocation')[0].get('longitude')
+        lat = data.get('geolocation')[0].get('latitude')
+        point = Point(lng, lat)
+        feat = Feature()
+        feat.project_id = projectId
+        feat.the_geom = from_shape(point, srid=4326)
+        feat.properties = metadata or data
+
+        asset_uuid = uuid.uuid4()
+        base_filepath = make_project_asset_dir(projectId)
+        asset_path = os.path.join(base_filepath, str(asset_uuid) + '.rq')
```

Comment:
should we place the rq (and eventually any assets) in a subfolder of the project?

Reply:
I think we should still have this in a subfolder like either one of the following:

- /assets/PROJECTEDID/UUID/UIUD.rq
- /assets/PROJECTEDID/UUID/questionaire.rq